rt/transports/http/pom.xml

```diff
@@ -281,4 +281,30 @@
             </plugins>
         </pluginManagement>
     </build>
+    
+    <!-- TODO: Remove once OSGi is supported or / and extracted -->
```

Comment:
@jimma unblocking `transport-http` by excluding the OSGi classes from compilation (temporary), wdyt?

Reply:
@reta  This looks good to me.  Thanks.